pkg/soperator/soperator.go

```diff
@@ -37,11 +36,6 @@ type SpecConfig struct {
 	Namespace                     string
 	Image                         string
 	Expose                        string
-	SecretType                    horizonapi.SecretType
```

Comment:
Are we removing the ability of the user to set secret type?  If so, and I think that's fine, we should remove at the cmdline option as well.

Reply:
Yup, it's already removed in [line 144](https://github.com/blackducksoftware/synopsys-operator/pull/329/files#diff-f1eb17c8864c31dbfd2ca3c0e7a8bde7L144) :-)